### DISCUSSION TITLE
1.2.0 beta 4 - no settings menu and lost focus

### DISCUSSION THREAD

**Discussion by andivax** [apohl79/audiogridder#674] [Help]
Hi guys!
Just installed 1.2.0 beta 4 and found 2 bugs

1) when launching vst2/vst3 audiogridder fx plugins, clicking on it not opens any menus like server or presets, etc
clicking anywhere on GUI is just not working

2) when loading plugins in inserts and clicking on plugin names and plugins opens behind DAW (not in focus)

I am using audiogridder on the same machine to free up my ASIO
here is the settings: https://prnt.sc/1gq06zs

**Comment by CaBuffalo** [comment]
Hi， what's your set up? Please provide more details if possible.

Simply saying not working doesn't help us to find out what the problem is.

Thanks.

On Tue, Jul 27, 2021 at 11:49 AM andivax ***@***.***> wrote:

> Hi guys!
> Just installed 1.2.0 beta 4 and found 2 bugs
>
>    1.
>
>    when launching vst2/vst3 audiogridder fx plugins, clicking on it not
>    opens any menus like server or presets, etc
>    clicking anywhere on GUI is just not working
>    2.
>
>    when loading plugins in inserts and clicking on plugin names and
>    plugins opens behind DAW (not in focus)
>
> I am using audiogridder on the same machine to free up my ASIO
> here is the settings: https://prnt.sc/1gq06zs
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/674>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AW4XK65N2YOA2FGKDTZ35U3ANCNFSM5BC2IEVQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by andivax** [comment]
Win 10 x64 21h1
Reaper 6 x64

Вт, 27 июля 2021 г. в 21:56, CaBuffalo ***@***.***>:

> Hi， what's your set up? Please provide more details if possible.
>
> Simply saying not working doesn't help us to find out what the problem is.
>
> Thanks.
>
> On Tue, Jul 27, 2021 at 11:49 AM andivax ***@***.***> wrote:
>
> > Hi guys!
> > Just installed 1.2.0 beta 4 and found 2 bugs
> >
> > 1.
> >
> > when launching vst2/vst3 audiogridder fx plugins, clicking on it not
> > opens any menus like server or presets, etc
> > clicking anywhere on GUI is just not working
> > 2.
> >
> > when loading plugins in inserts and clicking on plugin names and
> > plugins opens behind DAW (not in focus)
> >
> > I am using audiogridder on the same machine to free up my ASIO
> > here is the settings: https://prnt.sc/1gq06zs
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/apohl79/audiogridder/discussions/674>, or
> unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ARMAR6AW4XK65N2YOA2FGKDTZ35U3ANCNFSM5BC2IEVQ
> >
> > .
> >
>
>
> --
> Kevin Ma
> Supported by Gmail.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067421>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARLSQULHQGZBHRBC3OUWUSLTZ36NDANCNFSM5BC2IEVQ>
> .
>
-- 
www.ANDIVAXMASTERING.com

**Comment by CaBuffalo** [comment]
I see.

If you are not using screen capture, you won't be able to use the plugin
windows in Reaper. And yes, the plugin opens in another window outside the
DAW. That's why you say it's behind the DAW since you have the DAW
activated.

On Tue, Jul 27, 2021 at 12:01 PM andivax ***@***.***> wrote:

> Win 10 x64 21h1
> Reaper 6 x64
>
> Вт, 27 июля 2021 г. в 21:56, CaBuffalo ***@***.***>:
>
> > Hi， what's your set up? Please provide more details if possible.
> >
> > Simply saying not working doesn't help us to find out what the problem
> is.
> >
> > Thanks.
> >
> > On Tue, Jul 27, 2021 at 11:49 AM andivax ***@***.***> wrote:
> >
> > > Hi guys!
> > > Just installed 1.2.0 beta 4 and found 2 bugs
> > >
> > > 1.
> > >
> > > when launching vst2/vst3 audiogridder fx plugins, clicking on it not
> > > opens any menus like server or presets, etc
> > > clicking anywhere on GUI is just not working
> > > 2.
> > >
> > > when loading plugins in inserts and clicking on plugin names and
> > > plugins opens behind DAW (not in focus)
> > >
> > > I am using audiogridder on the same machine to free up my ASIO
> > > here is the settings: https://prnt.sc/1gq06zs
> > >
> > > —
> > > You are receiving this because you are subscribed to this thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/apohl79/audiogridder/discussions/674>, or
> > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6AW4XK65N2YOA2FGKDTZ35U3ANCNFSM5BC2IEVQ
> > >
> > > .
> > >
> >
> >
> > --
> > Kevin Ma
> > Supported by Gmail.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067421
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ARLSQULHQGZBHRBC3OUWUSLTZ36NDANCNFSM5BC2IEVQ
> >
> > .
> >
> --
> www.ANDIVAXMASTERING.com
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067446>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DH76BD5ABR4Q2NDYDTZ367PANCNFSM5BC2IEVQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by andivax** [comment]
Is it possible to use audiogridder without screen capture?
All my friends use it on the same machine to free up asio

Вт, 27 июля 2021 г. в 22:43, CaBuffalo ***@***.***>:

> I see.
>
> If you are not using screen capture, you won't be able to use the plugin
> windows in Reaper. And yes, the plugin opens in another window outside the
> DAW. That's why you say it's behind the DAW since you have the DAW
> activated.
>
> On Tue, Jul 27, 2021 at 12:01 PM andivax ***@***.***> wrote:
>
> > Win 10 x64 21h1
> > Reaper 6 x64
> >
> > Вт, 27 июля 2021 г. в 21:56, CaBuffalo ***@***.***>:
> >
> > > Hi， what's your set up? Please provide more details if possible.
> > >
> > > Simply saying not working doesn't help us to find out what the problem
> > is.
> > >
> > > Thanks.
> > >
> > > On Tue, Jul 27, 2021 at 11:49 AM andivax ***@***.***> wrote:
> > >
> > > > Hi guys!
> > > > Just installed 1.2.0 beta 4 and found 2 bugs
> > > >
> > > > 1.
> > > >
> > > > when launching vst2/vst3 audiogridder fx plugins, clicking on it not
> > > > opens any menus like server or presets, etc
> > > > clicking anywhere on GUI is just not working
> > > > 2.
> > > >
> > > > when loading plugins in inserts and clicking on plugin names and
> > > > plugins opens behind DAW (not in focus)
> > > >
> > > > I am using audiogridder on the same machine to free up my ASIO
> > > > here is the settings: https://prnt.sc/1gq06zs
> > > >
> > > > —
> > > > You are receiving this because you are subscribed to this thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/apohl79/audiogridder/discussions/674>, or
> > > unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6AW4XK65N2YOA2FGKDTZ35U3ANCNFSM5BC2IEVQ
> > > >
> > > > .
> > > >
> > >
> > >
> > > --
> > > Kevin Ma
> > > Supported by Gmail.
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067421
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ARLSQULHQGZBHRBC3OUWUSLTZ36NDANCNFSM5BC2IEVQ
> > >
> > > .
> > >
> > --
> > www.ANDIVAXMASTERING.com
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067446
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ARMAR6DH76BD5ABR4Q2NDYDTZ367PANCNFSM5BC2IEVQ
> >
> > .
> >
>
>
> --
> Kevin Ma
> Supported by Gmail.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1069725>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARLSQUIZGVLIEB7TECLDRELTZ4D5RANCNFSM5BC2IEVQ>
> .
>
-- 
www.ANDIVAXMASTERING.com

**Comment by CaBuffalo** [comment]
Yes, you can. Check the "Local mode" as shown in your screenshot and see if
it works for you.

On Tue, Jul 27, 2021 at 12:45 PM andivax ***@***.***> wrote:

> Is it possible to use audiogridder without screen capture?
> All my friends use it on the same machine to free up asio
>
> Вт, 27 июля 2021 г. в 22:43, CaBuffalo ***@***.***>:
>
> > I see.
> >
> > If you are not using screen capture, you won't be able to use the plugin
> > windows in Reaper. And yes, the plugin opens in another window outside
> the
> > DAW. That's why you say it's behind the DAW since you have the DAW
> > activated.
> >
> > On Tue, Jul 27, 2021 at 12:01 PM andivax ***@***.***> wrote:
> >
> > > Win 10 x64 21h1
> > > Reaper 6 x64
> > >
> > > Вт, 27 июля 2021 г. в 21:56, CaBuffalo ***@***.***>:
> > >
> > > > Hi， what's your set up? Please provide more details if possible.
> > > >
> > > > Simply saying not working doesn't help us to find out what the
> problem
> > > is.
> > > >
> > > > Thanks.
> > > >
> > > > On Tue, Jul 27, 2021 at 11:49 AM andivax ***@***.***> wrote:
> > > >
> > > > > Hi guys!
> > > > > Just installed 1.2.0 beta 4 and found 2 bugs
> > > > >
> > > > > 1.
> > > > >
> > > > > when launching vst2/vst3 audiogridder fx plugins, clicking on it
> not
> > > > > opens any menus like server or presets, etc
> > > > > clicking anywhere on GUI is just not working
> > > > > 2.
> > > > >
> > > > > when loading plugins in inserts and clicking on plugin names and
> > > > > plugins opens behind DAW (not in focus)
> > > > >
> > > > > I am using audiogridder on the same machine to free up my ASIO
> > > > > here is the settings: https://prnt.sc/1gq06zs
> > > > >
> > > > > —
> > > > > You are receiving this because you are subscribed to this thread.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/apohl79/audiogridder/discussions/674>, or
> > > > unsubscribe
> > > > > <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6AW4XK65N2YOA2FGKDTZ35U3ANCNFSM5BC2IEVQ
> > > > >
> > > > > .
> > > > >
> > > >
> > > >
> > > > --
> > > > Kevin Ma
> > > > Supported by Gmail.
> > > >
> > > > —
> > > > You are receiving this because you authored the thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067421
> > > >,
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ARLSQULHQGZBHRBC3OUWUSLTZ36NDANCNFSM5BC2IEVQ
> > > >
> > > > .
> > > >
> > > --
> > > www.ANDIVAXMASTERING.com
> > >
> > > —
> > > You are receiving this because you commented.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067446
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6DH76BD5ABR4Q2NDYDTZ367PANCNFSM5BC2IEVQ
> > >
> > > .
> > >
> >
> >
> > --
> > Kevin Ma
> > Supported by Gmail.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1069725
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ARLSQUIZGVLIEB7TECLDRELTZ4D5RANCNFSM5BC2IEVQ
> >
> > .
> >
> --
> www.ANDIVAXMASTERING.com
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1069849>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CGE465TT7SPYW6DY3TZ4EFFANCNFSM5BC2IEVQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by andivax** [comment]
thanx!
plugins on top and local mode is solved my first issue
but these buttons still is not clickable: https://prnt.sc/1gr0oue

вт, 27 июл. 2021 г. в 22:51, CaBuffalo ***@***.***>:

> Yes, you can. Check the "Local mode" as shown in your screenshot and see if
> it works for you.
>
> On Tue, Jul 27, 2021 at 12:45 PM andivax ***@***.***> wrote:
>
> > Is it possible to use audiogridder without screen capture?
> > All my friends use it on the same machine to free up asio
> >
> > Вт, 27 июля 2021 г. в 22:43, CaBuffalo ***@***.***>:
> >
> > > I see.
> > >
> > > If you are not using screen capture, you won't be able to use the
> plugin
> > > windows in Reaper. And yes, the plugin opens in another window outside
> > the
> > > DAW. That's why you say it's behind the DAW since you have the DAW
> > > activated.
> > >
> > > On Tue, Jul 27, 2021 at 12:01 PM andivax ***@***.***> wrote:
> > >
> > > > Win 10 x64 21h1
> > > > Reaper 6 x64
> > > >
> > > > Вт, 27 июля 2021 г. в 21:56, CaBuffalo ***@***.***>:
> > > >
> > > > > Hi， what's your set up? Please provide more details if possible.
> > > > >
> > > > > Simply saying not working doesn't help us to find out what the
> > problem
> > > > is.
> > > > >
> > > > > Thanks.
> > > > >
> > > > > On Tue, Jul 27, 2021 at 11:49 AM andivax ***@***.***> wrote:
> > > > >
> > > > > > Hi guys!
> > > > > > Just installed 1.2.0 beta 4 and found 2 bugs
> > > > > >
> > > > > > 1.
> > > > > >
> > > > > > when launching vst2/vst3 audiogridder fx plugins, clicking on it
> > not
> > > > > > opens any menus like server or presets, etc
> > > > > > clicking anywhere on GUI is just not working
> > > > > > 2.
> > > > > >
> > > > > > when loading plugins in inserts and clicking on plugin names and
> > > > > > plugins opens behind DAW (not in focus)
> > > > > >
> > > > > > I am using audiogridder on the same machine to free up my ASIO
> > > > > > here is the settings: https://prnt.sc/1gq06zs
> > > > > >
> > > > > > —
> > > > > > You are receiving this because you are subscribed to this thread.
> > > > > > Reply to this email directly, view it on GitHub
> > > > > > <https://github.com/apohl79/audiogridder/discussions/674>, or
> > > > > unsubscribe
> > > > > > <
> > > > >
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6AW4XK65N2YOA2FGKDTZ35U3ANCNFSM5BC2IEVQ
> > > > > >
> > > > > > .
> > > > > >
> > > > >
> > > > >
> > > > > --
> > > > > Kevin Ma
> > > > > Supported by Gmail.
> > > > >
> > > > > —
> > > > > You are receiving this because you authored the thread.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <
> > > >
> > >
> >
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067421
> > > > >,
> > > > > or unsubscribe
> > > > > <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ARLSQULHQGZBHRBC3OUWUSLTZ36NDANCNFSM5BC2IEVQ
> > > > >
> > > > > .
> > > > >
> > > > --
> > > > www.ANDIVAXMASTERING.com
> > > >
> > > > —
> > > > You are receiving this because you commented.
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1067446
> > > >,
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6DH76BD5ABR4Q2NDYDTZ367PANCNFSM5BC2IEVQ
> > > >
> > > > .
> > > >
> > >
> > >
> > > --
> > > Kevin Ma
> > > Supported by Gmail.
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1069725
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ARLSQUIZGVLIEB7TECLDRELTZ4D5RANCNFSM5BC2IEVQ
> > >
> > > .
> > >
> > --
> > www.ANDIVAXMASTERING.com
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1069849
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ARMAR6CGE465TT7SPYW6DY3TZ4EFFANCNFSM5BC2IEVQ
> >
> > .
> >
>
>
> --
> Kevin Ma
> Supported by Gmail.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/674#discussioncomment-1070197>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARLSQUPD7LBOHOVQEU3P3KDTZ4E23ANCNFSM5BC2IEVQ>
> .
>


-- 

<http://www.andivaxmastering.com/>

Andrii "ANDI VAX" Vakhnenko

www.ANDIVAXMASTERING.com <http://www.andivaxmastering.com/>

[image: Facebook] <https://www.facebook.com/groups/207215009301681/>   [image:
Youtube] <https://www.youtube.com/user/andivax>
